Src/DependencyCollector/NetCore/DependencyCollectorDiagnosticListener.cs

```diff
@@ -111,7 +110,9 @@ public void OnRequest(HttpRequestMessage request, Guid loggingRequestId)
         {
             try
             {
-                if (request != null && request.RequestUri != null && !this.applicationInsightsUrlFilter.IsApplicationInsightsUrl(request.RequestUri.ToString()))
+                if (request != null && request.RequestUri != null &&
+                    !this.applicationInsightsUrlFilter.IsApplicationInsightsUrl(request.RequestUri.ToString()) &&
+                    !this.correlationDomainExclusionList.Contains(request.RequestUri.Host))
```

Comment:
why are you mixing those two? `correlationDomainExclusionList` means "do not correlate", not "do not collect"